### PR TITLE
fix: fix flake8 errors in report module

### LINKED ISSUE #188: Fix flake8 issues in report module
It is a good idea to fix the remaining pep8 issues in the report module.
You can see the errors with 
```
tox -e flake8
```
These are currently for the report module mainly issues with the doc strings and line length/formating:
```c/sbmlutils/report/formating.py:1:1: D200 One-line docstring should fit on one line with quotes
src/sbmlutils/report/formating.py:10:1: D401 First line should be in imperative mood
src/sbmlutils/report/formating.py:39:1: D401 First line should be in imperative mood
src/sbmlutils/report/formating.py:48:1: D401 First line should be in imperative mood
src/sbmlutils/report/formating.py:85:1: D400 First line should end with a period
src/sbmlutils/report/formating.py:88:89: E501 line too long (93 > 88 characters)
src/sbmlutils/report/formating.py:89:89: E501 line too long (96 > 88 characters)
src/sbmlutils/report/formating.py:112:1: D400 First line should end with a period
src/sbmlutils/report/formating.py:112:1: D401 First line should be in imperative mood; try rephrasing
src/sbmlutils/report/formating.py:124:1: D400 First line should end with a period
src/sbmlutils/report/formating.py:149:1: D400 First line should end with a period
src/sbmlutils/report/formating.py:149:1: D401 First line should be in imperative mood; try rephrasing
src/sbmlutils/report/formating.py:172:89: E501 line too long (119 > 88 characters)
src/sbmlutils/report/formating.py:177:1: D401 First line should be in imperative mood; try rephrasing
src/sbmlutils/report/formating.py:196:1: D401 First line should be in imperative mood
src/sbmlutils/report/formating.py:228:1: D401 First line should be in imperative mood
src/sbmlutils/report/formating.py:251:1: D401 First line should be in imperative mood
src/sbmlutils/report/formating.py:281:1: D205 1 blank line required between summary line and description
src/sbmlutils/report/formating.py:281:1: D401 First line should be in imperative mood
src/sbmlutils/report/sbmlreport.py:2:89: E501 line too long (95 > 88 characters)
src/sbmlutils/report/sbmlreport.py:12:89: E501 line too long (95 > 88 characters)
src/sbmlutils/report/sbmlreport.py:41:1: D401 First line should be in imperative mood
src/sbmlutils/report/sbmlreport.py:47:89: E501 line too long (92 > 88 characters)
src/sbmlutils/report/sbmlreport.py:79:1: D401 First line should be in imperative mood
src/sbmlutils/report/sbmlreport.py:88:89: E501 line too long (92 > 88 characters)
src/sbmlutils/report/sbmlreport.py:147:89: E501 line too long (111 > 88 characters)
src/sbmlutils/report/sbmlreport.py:189:89: E501 line too long (111 > 88 characters)
src/sbmlutils/report/sbmlreport.py:251:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:329:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:348:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:360:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:362:89: E501 line too long (90 > 88 characters)
src/sbmlutils/report/sbmlreport.py:382:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:417:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:448:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:448:89: E501 line too long (91 > 88 characters)
src/sbmlutils/report/sbmlreport.py:464:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:479:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:496:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:523:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:572:89: E501 line too long (100 > 88 characters)
src/sbmlutils/report/sbmlreport.py:578:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:578:89: E501 line too long (96 > 88 characters)
src/sbmlutils/report/sbmlreport.py:599:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:615:89: E501 line too long (103 > 88 characters)
src/sbmlutils/report/sbmlreport.py:626:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:643:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:661:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:676:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:676:89: E501 line too long (89 > 88 characters)
src/sbmlutils/report/sbmlreport.py:712:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:740:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:753:89: E501 line too long (114 > 88 characters)
src/sbmlutils/report/sbmlreport.py:778:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:789:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:789:1: D401 First line should be in imperative mood
src/sbmlutils/report/sbmlreport.py:800:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:800:1: D401 First line should be in imperative mood
src/sbmlutils/report/sbmlreport.py:807:89: E501 line too long (120 > 88 characters)
src/sbmlutils/report/sbmlreport.py:812:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:812:1: D401 First line should be in imperative mood
src/sbmlutils/report/sbmlreport.py:830:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:836:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:847:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:860:89: E501 line too long (97 > 88 characters)
src/sbmlutils/report/sbmlreport.py:874:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:882:89: E501 line too long (101 > 88 characters)
src/sbmlutils/report/sbmlreport.py:890:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:902:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:909:89: E501 line too long (105 > 88 characters)
src/sbmlutils/report/sbmlreport.py:915:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:926:1: D401 First line should be in imperative mood
src/sbmlutils/report/sbmlreport.py:940:89: E501 line too long (145 > 88 characters)
src/sbmlutils/report/sbmlreport.py:945:89: E501 line too long (157 > 88 characters)
src/sbmlutils/report/sbmlreport.py:954:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:954:1: D401 First line should be in imperative mood
src/sbmlutils/report/sbmlreport.py:954:89: E501 line too long (98 > 88 characters)
src/sbmlutils/report/sbmlreport.py:967:1: D400 First line should end with a period
src/sbmlutils/report/sbmlreport.py:967:1: D401 First line should be in imperative mood
src/sbmlutils/report/xslt/mathml.py:1:1: D200 One-line docstring should fit on one line with quotes
```
This gives you a information on how to write good and correct doc strings with formating.

### PR DESCRIPTION
**Pull request by sankha555**
* [x] fix #188 
* [x] generated a test report for flake8 errors in reports module and fixed the errors in doc strings and formatting.
* [x] tests passed

### CODE REVIEW

**Comment by sankha555** [comment]
@matthiaskoenig I have made the changes in module according to the errors generated by the flake8 run. I changed the function summary description in a few places to make it a bit concise.

In some of the strings in ```sbmlreport.py```, where the HTML code fragments are rendered, I have indented those strings into multiline strings since some of them were crossing the character limit per line as indicated by flake8. Would this change be appropriate in terms of standards and readability?  

I am a bit confused about what best change I should incorporate for a few instances in the files, which I have asked in the comments in the corresponding line. Please take a look at them and kindly suggest me how I should proceed regarding them. If there are any other changes that I should make, please let me know. Thanks!

File: src/sbmlutils/report/formating.py [hunk 1]
```diff
@@ -169,12 +169,17 @@ def boundsStringFromReaction(reaction: libsbml.Reaction, model: libsbml.Model) -
             if ub_p.isSetValue():
                 ub_value = ub_p.getValue()
         if (lb_value is not None) or (ub_value is not None):
-            bounds = f'<code>[{lb_value} <i class="fa fa-sort fa-rotate-90" aria-hidden="true"></i> {ub_value}]</code>'
+            bounds = f"""
```
Comment (sankha555) [inline src/sbmlutils/report/formating.py:172]: One of the instances, where I have made the HTML code multi-line to overcome the flake8 error

Comment (sankha555) [reply src/sbmlutils/report/formating.py:172]: In the first doc string for the files ```xslt/math.py``` (line 2) and ```formating.py```(line 2), flake8 says that they should be single line doc-strings. Should I convert them to single line doc strings or shall Iet them be as they are. If we need to update those doc strings in the future to add more detail, then we might have to convert them to multi-line doc strings. Hence I wanted to take your suggestion on what would be best way to go about them.

Comment (matthiaskoenig) [reply src/sbmlutils/report/formating.py:172]: xlst will be removed soon, so no updates here. Multi-line comment is fine.

**Pull request merged**